@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -131,102 +131,106 @@ function createTextureFromRawImage(
     }
 }
 
+function initTextureProperties(texture: THREE.Texture, properties?: TextureProperties) {
+    if (!properties) {
+        return;
+    }
+    if (properties.wrapS !== undefined) {
+        texture.wrapS = toWrappingMode(properties.wrapS);
+    }
+    if (properties.wrapT !== undefined) {
+        texture.wrapT = toWrappingMode(properties.wrapT);
+    }
+    if (properties.magFilter !== undefined) {
+        texture.magFilter = toTextureFilter(properties.magFilter);
+    }
+    if (properties.minFilter !== undefined) {
+        texture.minFilter = toTextureFilter(properties.minFilter);
+    }
+    if (properties.flipY !== undefined) {
+        texture.flipY = properties.flipY;
+    }
+    if (properties.repeatU !== undefined) {
+        texture.repeat.x = properties.repeatU;
+    }
+    if (properties.repeatV !== undefined) {
+        texture.repeat.y = properties.repeatV;
+    }
+}
+
 function createTexture(
```

Comment:
consider to mark as async

Reply:
It's actually a synchronous function, marking it async is actually misleading.

As for making it asynchronous, we don't want to await on the texture here. We need to trigger the loading of all textures in the tile and later wait for all of them to be resolved concurrently.